packages/components/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@hyparam/components",
+  "version": "0.1.0",
+  "description": "React components for hyparam apps",
+  "keywords": [
+    "component",
+    "data",
+    "javascript",
+    "react"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyparam/hyperparam-cli.git",
+    "directory": "packages/components"
+  },
+  "type": "module",
+  "main": "dist/main.umd.js",
+  "module": "dist/main.es.js",
+  "exports": {
+    ".": {
+      "import": "./dist/main.es.js",
+      "require": "./dist/main.umd.js",
+      "types": "./dist/main.d.ts"
+    }
+  },
+  "types": "dist/main.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -b ./tsconfig.lib.json && vite build",
+    "coverage": "vitest run --coverage --coverage.include=src",
+    "demo": "vite",
+    "lint": "eslint .",
+    "preview": "vite preview",
+    "prepublishOnly": "npm run build",
+    "test": "vitest run"
+  },
+  "peerDependencies": {
+    "hightable": "0.6.3",
+    "hyparquet": "1.5.0",
+    "hyparquet-compressors": "0.1.4",
```

Comment:
These should probably be `dependencies` not `peerDependencies`? When bundled I would think we want to include hightable, etc. Whereas react is likely already included by any project using these components?

Reply:
done